### PR TITLE
CHAOS-6233 FIS Generic experiment template docs

### PR DESCRIPTION
**Pull request by SmritiSatya**
Generic experiment template docs
Fault diagram
Sample manifest

### CODE REVIEW

**Review by jenniferopal — DISMISSED** [review]
lgtm!

File: docs/chaos-engineering/chaos-faults/aws/aws.md [hunk 1]
```diff
@@ -718,6 +734,26 @@ ECS update task role allows you to modify the IAM task role associated with an A
 
 <FaultDetailsCard category="aws">
 
+### Generic experiment template
+
+Generic experiment template provides a template to natively inject faults using FIS for different services, such as EC2, EBS, DynamoDB, and so on.
+- You need to create an FIS template and store it.
```
Comment (uditgaurav) [inline docs/chaos-engineering/chaos-faults/aws/aws.md:740]:
```suggestion
- You need to create an FIS template.
```

File: docs/chaos-engineering/chaos-faults/aws/aws.md [hunk 2]
```diff
@@ -718,6 +734,26 @@ ECS update task role allows you to modify the IAM task role associated with an A
...
+- Provide parameters to the pre-created FIS templates and execute experiments.
+- You can specify the template ID and region on Harness to execute the experiments using these FIS templates.
```
Comment (uditgaurav) [inline docs/chaos-engineering/chaos-faults/aws/aws.md:742]:
```suggestion
- You can specify the template ID, region, and parameters on Harness to execute the experiments using these FIS templates.
```

File: docs/chaos-engineering/chaos-faults/aws/generic-experiment-template.md [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+      <td> RAMP_TIME </td>
+      <td> Period to wait before and after injection of chaos in sec </td>
+      <td> For example, 30 s. For more information, go to <a href="/docs/chaos-engineering/chaos-faults/common-tunables-for-all-faults#ramp-time"> ramp time. </a></td>
+    </tr>
+    <tr>
+      <td> AWS_SHARED_CREDENTIALS_FILE </td>
+      <td> Path to the AWS secret credentials. </td>
+      <td> Default: <code>/tmp/cloud_config.yml</code>. </td>
+      </tr>
+  </table>
```
Comment (uditgaurav) [inline docs/chaos-engineering/chaos-faults/aws/generic-experiment-template.md:120]: add "parameters" as the optional variable

**Review by uditgaurav — DISMISSED** [review]

**Pull request merged**